lib/handler.rb

```diff
@@ -0,0 +1,19 @@
+class Handler
```

Comment:
Why do you have all of these `NotImplementedError`s? Can you just delete this class?